### PR TITLE
Enhance issue templates and issue choices

### PR DESCRIPTION
**Pull request by MichaelSimons**
1. Added a `Feature request` issue template.
2. Refactored the `Bug report` issue template.
3. Added an issue config file with entries for the common types of issues .NET Docker users encounter.
4. Simplified the `Feedback` section of the readmes to contain one `File an issue` link which takes users to the new issue selection page of this repo.

You can see how the new issue page looks in [my fork](https://github.com/MichaelSimons/dotnet-docker/issues/new/choose).

[skip ci]

### CODE REVIEW

File: README.aspnet.md [hunk 1]
```diff
@@ -118,13 +118,7 @@ See [Microsoft Support for .NET Core](https://github.com/dotnet/core/blob/master
...
 
-* [File a .NET Core Docker issue](https://github.com/dotnet/dotnet-docker/issues)
-* [File a .NET Core issue](https://github.com/dotnet/core/issues)
-* [File an ASP.NET Core issue](https://github.com/aspnet/home/issues)
-* [File an issue for other .NET components](https://github.com/dotnet/core/blob/master/Documentation/core-repos.md)
-* [File a Visual Studio Docker Tools issue](https://github.com/microsoft/dockertools/issues)
-* [File a Microsoft Container Registry (MCR) issue](https://github.com/microsoft/containerregistry/issues)
-* [Ask on Stack Overflow](https://stackoverflow.com/questions/tagged/.net-core)
+* [File an issue](https://github.com/dotnet/dotnet-docker/issues/new/choose)
 * [Contact Microsoft Support](https://support.microsoft.com/contactus/)
```
Comment (MichaelSimons) [inline README.aspnet.md:122]: Thoughts on including this as one of the dotnet-docker new issue options and removing it here?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Questions
```
Comment (MichaelSimons) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Once the GH discussions feature is released, I propose we enable discussions in this repo and direct users to it versus Stack Overflow.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -1,31 +1,45 @@
...
-## Steps to reproduce the issue
-1.
-2.
-3.
+<!--
+Please keep in mind that the GitHub issue tracker is not intended as a general support forum, but for reporting **non-security** bugs and feature requests.
+
+If you believe you have an issue that affects the SECURITY of the platform, please do NOT create an issue and instead email your issue details to secure@microsoft.com. Your report may be eligible for our [bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) but ONLY if it is reported through email.
+
+For other types of questions, consider using [StackOverflow](https://stackoverflow.com).
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/bug-report.md:12]:
```suggestion
For other types of questions, consider using [Stack Overflow](https://stackoverflow.com).
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -1,31 +1,45 @@
...
+
+### Describe the Bug
+
+<!-- A clear and concise description of what the bug is. -->
 
-## Expected behavior
+### Steps to Reproduce
 
+<!--
+We ❤ code! Include minimal steps to reproduce the problem if possible or point us to a minimalistic repro project hosted in a GitHub repo.
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/bug-report.md:22]:
```suggestion
We ❤ code! Include minimal steps to reproduce the problem if possible or point us to a simple repro project hosted in a GitHub repo.
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -1,31 +1,45 @@
...
 
-## Actual behavior
+We will close this issue if:
+- the repro project you share with us is complex. We can't investigate complex projects, so don't point us to such, please.
+- if we are not be able to repro the behavior you're reporting
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/bug-report.md:26]:
```suggestion
- if we are not able to repro the behavior you're reporting
```

File: README.aspnet.md (hunk 1 above)
Comment (mthalman) [reply README.aspnet.md:122]: I think leave it here.  Filing an issue and requesting support are different things.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  - name: Issue with ASP.NET Core
+    url:  https://github.com/dotnet/efcore/issues/new/choose
+    about: Please open issues relating to ASP.NET Core in dotnet/aspnetcore.
+  - name: Issue with .NET SDK
+    url:  https://github.com/dotnet/sdk/issues/new/choose
+    about: Please open issues relating to the .NET SDK itself in dotnet/sdk.
+  - name: Issue with other .NET components
+    url:  https://github.com/dotnet/core/blob/master/Documentation/core-repos.md
+    about: Having issues with other .NET components? Please find the correct location where to open an issue.
+  - name: Issue with Visual Studio docker tools
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/config.yml:18]:
```suggestion
  - name: Issue with Visual Studio Docker Tools
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    url:  https://github.com/microsoft/dockertools/issues
+    about: Please open issues relating to the Visual Studio docker tooling in microsoft/dockertools.
```
Comment (mthalman) [inline .github/ISSUE_TEMPLATE/config.yml:20]:
```suggestion
    about: Please open issues relating to the Visual Studio Docker tooling in microsoft/dockertools.
```

Comment (MichaelSimons) [reply .github/ISSUE_TEMPLATE/config.yml:20]: I am changing the about and name to `Visual Studio Tools for Docker` to align with the name the product uses.

**Review by mthalman — APPROVED** [review]

**Pull request merged**